vsintegration/src/vs/FsPkgs/FSharp.VS.FSI/fsiSessionToolWindow.fs

```diff
@@ -587,13 +590,16 @@ type internal FsiToolWindow() as this =
                  // REVIEW: log error into Trace.
                  // Example errors include no active document.
 
-    let onMLSend (sender:obj) (e:EventArgs) =       
-        sendSelectionToFSI false
+    let onMLSendSelection (sender:obj) (e:EventArgs) =       
+        sendSelectionToFSI false false
```

Comment:
As I said above, `sendSelectionToFSI false false` is very hard to guess. We can't even add parameter names here.